marklogic-data-hub/src/main/java/com/marklogic/hub/step/impl/WriteStepRunner.java

```diff
@@ -539,6 +539,15 @@ private RunStepResponse runIngester(RunStepResponse runStepResponse, Collection<
             while(itr.hasNext()) {
                 try {
                     File file = new File((String) itr.next());
+                    if("csv".equalsIgnoreCase(inputFileType)) {
+                        //Add "fileName" to options so that it gets added to header in enode code
+                        options.put("fileName", file.getAbsolutePath());
+                        optionString = jsonToString(options);
+                        serverTransform.remove("options");
+                        serverTransform.addParameter("options", optionString);
+                        //Add additional document metadata
+                        metadataValues.add("datahubCreatedUsingFile" , file.getAbsolutePath());
```

Comment:
Since only "File" is used here, then I think that suggests using just "file" above for consistency.